pkg/virt-controller/watch/migration.go

```diff
@@ -704,8 +707,29 @@ func (c *MigrationController) handleTargetPodCreation(key string, migration *vir
 	// migration was accepted into the system, now see if we
 	// should create the target pod
 	if vmi.IsRunning() {
-		if err := c.createPDB(key, vmi, migration); err != nil {
-			return err
+		if migrations.MigrationNeedsProtection(vmi) {
+			pdbs, err := pdbs.PDBsForVMI(vmi, c.pdbInformer)
+			if err != nil {
+				return err
+			}
+			if len(pdbs) < 1 {
+				log.Log.Object(vmi).Errorf("Found no PDBs protecting the vmi")
+				return err
+			}
+			pdb := pdbs[0]
+
+			if expandedPdb, err := c.expandPDB(pdb, vmi, migration); err != nil {
+				return err
+			} else {
+				pdb = expandedPdb
+			}
+
+			// before proceeding we have to check that the k8s pdb controller has processed
+			// the pdb expansion and is actually protecting the VMI migration
+			if !c.isMigrationProtected(pdb) {
+				log.Log.V(4).Object(migration).Infof("Kubernetes is still not protecting the VMI migration, postponing migration start")
```

Comment:
can we change the message here slightly and indicate that we're `waiting` on something specific `pdb controller` to protect the migration pods.

Reply:
Sure thing.